packages/client/hmi-client/src/components/widgets/share-project/tera-share-project.vue

```diff
@@ -14,8 +14,9 @@
 				:options="usersMenu"
 				optionLabel="name"
 				editable
+				showClear
 				placeholder="Add people and groups"
-				@update:model-value="(value) => addNewSelectedUser(value.id)"
+				@update:model-value="(value) => addNewSelectedUser(value?.id)"
```

Comment:
Why would `value` be null or defined here?

Reply:
I added the primevue clear button in the input field. When the clear button is used the value can be null.
I can remove it if you want
https://primevue.org/dropdown/#clearicon

Reply:
That sounds weird ... but okay, not going to make a big deal out of it 😄